src/main/java/zeus/main/TaskList.java

```diff
@@ -0,0 +1,103 @@
+package zeus.main;
+
+import java.util.ArrayList;
+
+import zeus.task.Task;
+
+
+/**
+ * Class that represents a list of tasks.
+ */
+public class TaskList {
+
+    private ArrayList<Task> taskList;
+
+    /**
+     * Constructor of Tasklist class.
+     */
+    public TaskList() {
+        this.taskList = new ArrayList<>();
+    }
+
+    public TaskList(ArrayList<Task> taskList) {
+        this.taskList = taskList;
+    }
+
+    /**
+     * Returns current size of task list.
+     *
+     * @return size of task list
+     */
+    public int size() {
```

Comment:
```suggestion
    public int getSize() {
```
Would it be better for this method name to be of a verb?